test/gpu/native/localeName.chpl

```diff
@@ -0,0 +1,4 @@
+writeln(here.name); // why empty?
```

Comment:
Really, this returns the empty string? Not that I think you need to fix it for this PR but do we get the same thing with `CHPL_LOCALE_MODEL=flat`?

Reply:
Nope. I get the hostname.